common_utils/generate_metric_dict.py

```diff
@@ -0,0 +1,22 @@
+# KLD and Jeffreys divergence are measures used to quantify the difference between two probability distributions. Why do we calculate these metrics in the context of forecasting?
+# There are negative values, so errors occur when calculating MSLE
+# Brier score is used for binary and categorical outcomes that can be structured as true or false
+# There are no classes in data, so we cannot calculate roc_auc_score, ap_score
+import properscoring as ps
+from sklearn.metrics import mean_squared_error, mean_absolute_error, mean_squared_log_error, brier_score_loss, average_precision_score, roc_auc_score
+from utils_evaluation_metrics import EvaluationMetrics
+from views_forecasts.extensions import *
+
+def generate_metric_dict(df, steps, depvar):
+    evaluation_dict = EvaluationMetrics.make_evaluation_dict(steps=steps[-1])
```

Comment:
I need a docstring here. 
I'm unsure which df it takes, but I suspect one with predictions and true etc.
You could of course know this last week, but we need to align that with the new data class for predictions. Bu we can do that later as long as it goes firmly on a tode list.

Additionally, if this is to be general it should take all to metrics defined in the workshop. But of course only fill in the ones relevant for the specific model. 

I have similar code in the PR for aligning eval and abstracting data loaders so we can try to align later.

Point: No need to do stuff now, but this should go on the todo list